llvm/lib/Target/RISCV/GISel/RISCVCallLowering.cpp

```diff
@@ -48,10 +51,16 @@ struct RISCVOutgoingValueAssigner : public CallLowering::OutgoingValueAssigner {
     const DataLayout &DL = MF.getDataLayout();
     const RISCVSubtarget &Subtarget = MF.getSubtarget<RISCVSubtarget>();
 
+    std::optional<unsigned> FirstMaskArgument;
+    if (Subtarget.hasVInstructions() && !AssignedFirstMaskArg &&
+        ValVT.isVector() && ValVT.getVectorElementType() == MVT::i1) {
+      FirstMaskArgument = std::make_optional(ValNo);
```

Comment:
Why do we need std::make_optional? Can't we do `FirstMaskArgument = ValNo`?

Reply:
updated.